timemachine/fe/topology.py

```diff
@@ -585,13 +584,13 @@ def parameterize_nonbonded(self, ff_q_params, ff_lj_params):
         # worth of nonbonded interactions.
 
         # dst_qlj_params corresponds to the end-state where only one of the molecule interacts with the binding pocket.
-        src_qlj_params_a = jax.ops.index_update(qlj_params_a, jax.ops.index[:, 0], qlj_params_a[:, 0] * 0.5)
-        src_qlj_params_a = jax.ops.index_update(src_qlj_params_a, jax.ops.index[:, 2], src_qlj_params_a[:, 2] * 0.5)
+        src_qlj_params_a = jnp.asarray(qlj_params_a).at[:, 0].set(qlj_params_a[:, 0] * 0.5)
+        src_qlj_params_a = jnp.asarray(src_qlj_params_a).at[:, 2].set(src_qlj_params_a[:, 2] * 0.5)
         dst_qlj_params_a = qlj_params_a
 
         qlj_params_b = standard_qlj_typer(self.mol_b)
-        src_qlj_params_b = jax.ops.index_update(qlj_params_b, jax.ops.index[:, 0], qlj_params_b[:, 0] * 0.5)
-        src_qlj_params_b = jax.ops.index_update(src_qlj_params_b, jax.ops.index[:, 2], src_qlj_params_b[:, 2] * 0.5)
+        src_qlj_params_b = jnp.asarray(qlj_params_b).at[:, 0].set(qlj_params_b[:, 0] * 0.5)
+        src_qlj_params_b = jnp.asarray(src_qlj_params_b).at[:, 2].set(src_qlj_params_b[:, 2] * 0.5)
```

Comment:
nit: these can be simplified slightly `x.at[idx].set(x[idx] * 0.5)` -> `x.at[idx].multiply(0.5)`